src/custom/pages/Swap/index.tsx

```diff
@@ -8,12 +8,7 @@ import { ButtonSize, TYPE } from 'theme/index'
 
 import SwapMod from './SwapMod'
 import { AutoRow, RowBetween, RowFixed } from 'components/Row'
-import {
-  BottomGrouping as BottomGroupingUni,
-  Wrapper as WrapperUni,
-  ArrowWrapper as ArrowWrapperUni,
-  Dots
-} from 'components/swap/styleds'
+import { BottomGrouping as BottomGroupingUni, Wrapper as WrapperUni, Dots } from 'components/swap/styleds'
```

Comment:
comment out the ArrowWrapper

Reply:
Not sure I understand: I removed `ArrowWrapper as ArrowWrapperUni,` because it was only used for having a targeting CSS selector. That is no longer in use. Commenting out wouldn't be necessary given this is not the Mod file?

Reply:
but i think it's their code no? or we added it? if it's the latter then yeah all good 👍